portfolio/src/main/webapp/script.js

```diff
@@ -56,23 +56,49 @@ function loadSkills () {
  */
 async function fetchFromData() {
   const response = await fetch('/data');
+  
   // This gives a list of messages
-  const msgs = await response.json();
+  const comments = await response.json();
 
   const root = document.getElementById("all-messages");
 
   // Adds all of the messages to a div
-  msgs.forEach(m => {
-    const node = document.createElement("p"); 
-    node.className = "msg"; 
-    node.textContent = m;
-    root.appendChild(node); 
+  comments.forEach(c => {
+    root.appendChild(createCommentElement(c)); 
   }); 
 
   $("#get-msg-btn").addClass("invisible"); 
   $("#hide-msg-btn").removeClass("invisible"); 
 };
 
+/**
+ * Creates a comment element with the content, name of commenter, and date
+ */
+function createCommentElement(comment) {
+  const node = document.createElement("div");
```

Comment:
why not use jquery for these dom manipulations :)

Reply:
If I'm being completely honest that was because I was more familiar with creating elements with the dom but changed it so now I know how to do it both ways!